static/styles/portico/stats.scss

```diff
@@ -218,6 +173,51 @@ hr {
     margin-left: 10px;
 }
 
+#id_messages_sent_by_client {
+    min-height: 100px;
+    width: 750px;
+    position: relative;
+}
+
+#id_messages_sent_by_message_type {
+    height: 300px;
+    width: 750px;
+    position: relative;
+}
+
+#id_messages_sent_over_time {
+    height: 400px;
+    width: 750px;
+    position: relative;
+}
+
+#id_messages_sent_over_time[last_value_is_partial='true'] .points path:last-of-type {
+    opacity: 0.5;
+}
+
+#id_number_of_users {
+    height: 370px;
+    width: 750px;
+    position: relative;
+}
+
+#users_hover_info,
+#hoverinfo {
+    display: none;
+    font-size: 0.8em;
+    font-weight: 400;
+    position: relative;
+    height: 0px;
+    top: -35px;
+    left: 40px;
+}
+
+@keyframes spinner {
```

Comment:
This should stay next to the `spinner` definition, I think.

Reply:
The animation definition can be declared anywhere on the page, so I thought to reorders the SCSS to go for elements, class, id, animation and media query in that order so one can quickly look for the animations in future if it needs modification.